client/src/main/java/no/nordicsemi/android/kotlin/ble/client/main/service/ClientBleGattService.kt

```diff
@@ -61,7 +61,7 @@ data class ClientBleGattService internal constructor(
      */
     val uuid = service.uuid
 
-    val characteristics = service.characteristics.map {
+    private val characteristics = service.characteristics.map {
```

Comment:
We have a feature where we displays all of the characteristics. It needs to be public.

Reply:
Ok sure, Android studio was complaining that this was not in use. You may add a suppression for this warning :)

Reply:
Done